ckan/model/system_info.py

```diff
@@ -1,29 +1,45 @@
 from sqlalchemy import types, Column, Table
 
+import vdm.sqlalchemy
 import meta
 import core
 import domain_object
 
 __all__ = ['system_info_revision_table', 'system_info_table', 'SystemInfo',
```

Comment:
It would be good to state clearly here that "SystemInfo" = "run-time editable config option"